src/contracts/interfaces/IRewardsCoordinator.sol

```diff
@@ -93,6 +94,18 @@ interface IRewardsCoordinator {
         uint32 duration;
     }
 
+    /// @notice Used for performance based reward submissions.
+    struct PerformanceRewardsSubmission {
```

Comment:
Can we also add natspec for each of these. Need to describe what scores & operators are.

Reply:
uint32[] scores specify how the AVS subjectively values each operator's performance. 

strategies[] specify which strategies to reward given stakers of an operator. Note: we likely have to change this to `StrategiesAndMultipliers` to be able to calculate the `stakeWeight` of a staker for performance based rewards.

Reply:
I tried adding natspec, I'm just not sure its correct. Might be a good idea for you to add docs since you're more familiar with off-chain.